buildkite/scripts/publish-deb.sh

```diff
@@ -31,8 +31,48 @@ esac
 echo "Publishing debs: ${DEBS}"
 set -x
 ${DEBS3} --codename "${CODENAME}" "${DEBS}"
+set +x
 echo "Exporting Variables: "
-# Export Variables for Downstream Steps
+
+GITHASH=$(git rev-parse --short=7 HEAD)
```

Comment:
This environment setup seems to be duplicated in `rebuild-deb`. Can we move it to a separate file that they both source/call, so that we know they'll be kept in sync?

Reply:
we've had that number get out of sync before too